### PR TITLE
Capture core dump for zos in post test step

### PR DESCRIPTION
**Pull request by JennyChenyj**
- Added moveTDUMP to move core dump to
TESTOUTPUT/<testname>

Signed-off-by: Jenny Chen <Jenny.Chen@ibm.com>

### CODE REVIEW

**Comment by JennyChenyj** [comment]
Hey @llxia! Thanks for checking it again!

File: scripts/moveDmp.pl [hunk 1]
```diff
@@ -0,0 +1,246 @@
+##############################################################################
+#  Licensed Materials - Property of IBM
+#  Copyright (c) 2006, 2019 IBM Corp. All Rights Reserved.
+#  US Government Users Restricted Rights - Use, duplication or
+#  disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+##############################################################################
```
Comment (llxia) [inline scripts/moveDmp.pl]: Please change the license to be the same as https://github.com/AdoptOpenJDK/TKG/blob/master/scripts/resultsSum.pl

Comment (JennyChenyj) [reply scripts/moveDmp.pl]: > Please change the license to be the same as https://github.com/AdoptOpenJDK/TKG/blob/master/scripts/resultsSum.pl

Sure. Thanks for the link~:)

**Comment by JennyChenyj** [comment]
Hey @llxia ! I simplified the code. Thanks for reviewing it!

**Pull request merged**